docs/graphql/manual/migrations/manage-migrations.rst

```diff
@@ -1,7 +1,7 @@
 .. _postgres_schema_metadata:
 
-Managing Postgres schema and Hasura metadata
-============================================
+Managing Postgres schema and Hasura migrations
```

Comment:
metadata is more accurate here :). typically we use the term migrations for the postgres schema while hasura has metadata. its all a little confusing and the lines are quite blurry but lets try to keep the terminology in docs consistent. This section is planned to be updated anyway to avoid the multiple docs paths for setting up migrations